Header/Header.module.less

```diff
@@ -46,6 +46,16 @@
 			text-indent: @agate-header-title-text-indent;  // Fix for characters like `j` being left-truncated because of being too close to the edge
 		}
 
+		.subtitle {
+			color: @agate-header-subtitle-text-color;
+			opacity: @agate-header-subtitle-opacity;
+			padding-top: @agate-header-subtitle-padding;
```

Comment:
This appears to work now only because the padding value is set to a single value, not a shorthand multi-value. This rule should switch to just plain `padding` and the `@agate-header-subtitle-padding` variable should become `9px 0 0 0`.